rustls/src/tls13/mod.rs

```diff
@@ -33,12 +38,11 @@ pub(crate) static TLS13_CHACHA20_POLY1305_SHA256_INTERNAL: &Tls13CipherSuite = &
         hash_provider: &crypto::ring::hash::SHA256,
     },
     hmac_provider: &crypto::ring::hmac::HMAC_SHA256,
-    aead_alg: &AeadAlgorithm(&ring::aead::CHACHA20_POLY1305),
+    aead_alg: &CHACHA20_POLY1305_AEAD,
```

Comment:
It became necessary only in this commit, because this is where I broke the compiler by adding one parameter to `AeadAlgorithm`, ie `&AeadAlgorithm(&ring::aead::CHACHA20_POLY1305, chacha20_poly1305_extract_keys)`